### PR TITLE
implement Dedication Ceremony and Heritage Committee

### PR DESCRIPTION
**Pull request by JoelCFC25**
New implementations for Dedication Ceremony and Heritage Committee. Also includes a small update to Chatterjee University to account for the pack-specific Unofficial FAQ that came out indicating you can use it to install a program when it has 0 counters.

### CODE REVIEW

File: src/clj/game/cards-resources.clj [hunk 1]
```diff
@@ -77,14 +77,15 @@
    {:abilities [{:cost [:click 1] :label "Place 1 power counter"
                  :msg "place 1 power counter on it"
                  :effect (effect (add-prop card :counter 1))}
-                {:req (req (> (get card :counter 0) 0))
+                {:req (req (>= (get card :counter 0) 0))
```
Comment (nealterrell) [inline src/clj/game/cards-resources.clj]: This function is always true ;).

Comment (JoelCFC25) [inline src/clj/game/cards-resources.clj]: So I just need to take it out?

Comment (nealterrell) [inline src/clj/game/cards-resources.clj]: Yeah. In effect there is no requirement for using the ability.

**Comment by mtgred** [comment]
:+1:

**Pull request merged**